develop/tests/API/v2/OrderTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+
+use Tests\Support\DatabaseTestCase;
+
+class OrderTest extends DatabaseTestCase{
+
+    public function setUp(): void
+	{
+		parent::setUp();
+
+		// Extra code to run before each test
+	}
+
+    public function tearDown(): void
+	{
+		parent::tearDown();
+
+        $this->db->table('db_order')->emptyTable('db_order');
```

Comment:
Do you command `./vendor/bin/php-cs-fixer fix` to these files?
If you really do, then why this line doesn't have indent?

Reply:
This change is done.